.swm/getting-started-with-world-engine.pxtvf.sw.md

```diff
@@ -1,8 +1,8 @@
 ---
 id: pxtvf
```

Comment:
Can you pull in the .swm changes from this PR: https://github.com/Argus-Labs/world-engine/pull/166/files#diff-d957e073d15e0bf36b576b3746ce31633d88e060524bedb96b779abc2127e871